src/pages/TodayPage/Components/NewTodayCard.tsx

```diff
@@ -0,0 +1,112 @@
+import { Flex, Input, Spacer, Text } from "@chakra-ui/react";
+import { useState } from "react";
+import CouchIcon from "../../../icons/CouchIcon";
+import DeleteIcon from "../../../icons/DeleteIcon";
+import NewTodayIcon from "../../../icons/NewTodayIcon";
+import NextWeekIcon from "../../../icons/NextWeekIcon";
+import SunIcon from "../../../icons/SunIcon";
+import UpsentIcon from "../../../icons/UpsentIcon";
+
+function NewTodayCard() {
+  const date = new Date(Date.now());
+  const weeksDate = new Date(Date.now() + 3600 * 1000 * 24);
+  const nextWeeks = new Date(Date.now() + 3600 * 1000 * 24 * 7);
+  const options3: Intl.DateTimeFormatOptions = {
+    weekday: "short",
+    day: "numeric",
+    month: "short",
+  };
+  const options2: Intl.DateTimeFormatOptions = {
+    weekday: "short",
+  };
+  const options1: Intl.DateTimeFormatOptions = {
+    month: "short",
+    day: "numeric",
+  };
+  const today = date.toLocaleDateString("en-US", options1);
+  const tomorrow = weeksDate.toLocaleDateString("en-US", options2);
+  const nextWeek = nextWeeks.toLocaleDateString("en-US", options3);
+  const [todays, setTodays] = useState(today);
+  const deleteHandler = (e: any) => {
+    e.preventDefault();
+    setTodays("");
+  };
+
+  return (
+    <Flex
+      borderRadius='2%'
+      border='1px solid'
+      borderColor='gray.200'
+      direction='column'
+      ml='3'
+      w='280px'
+      bg='white'
+    >
+      <Flex p='2'>
+        <Input
```

Comment:
I could not understand this part. Why did you add this Input? What place are you using it?

Reply:
In the original of the page, the input and the placaholder are seen when deleted.